cfpb/consumerfinance.gov
# Add fixed-position projected data label to charts

We've been relying on a "plot line" label which draws a vertical plot
line and labels it with some text. Unfortunately, if a user zooms to an
area of the chart that doesn't contain the plot line, the label is also
removed. We want the projected data label to always be visible so we're
now rendering it as text positioned over the chart.

## Additions

- Highcharts `render()` callback that draws some text whenever the chart is redrawn.

## How to test this PR

1. Pull down this branch
2. `yarn build`
3. Create a simple chart with projected months and it should have the standard `Values after XXXX are projected` text on the chart.


## Screenshots

<img width="687" alt="image" src="https://user-images.githubusercontent.com/1060248/169767445-86834560-a7c9-4497-9676-d793045a2b79.png">

## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
